src/Utility.js

```diff
@@ -92,6 +92,18 @@ const __class = lang.setObject('crm.Utility', lang.mixin({}, Utility, {
     }
     return id;
   },
+  trimText: function trimText(text = {}, wordCount = {}) {
+    const words = text.split(' ');
+    if (words.length > wordCount) {
+      const intermediate = words.slice(0, wordCount);
+      if (intermediate[wordCount - 1].slice(-1) === '.') {
+        intermediate[wordCount - 1] = intermediate[wordCount - 1].slice(0, -1);
+      }
+      const value = intermediate.join(' ') + ' ...';
```

Comment:
Yes but with the space between ' ...'  make it harder read with the dollar sing and brackets, is there advantage